src/sagas/transactions.js

```diff
@@ -0,0 +1,11 @@
+import { call } from 'redux-saga/effects';
+
+import { getPangeaLibrary } from '../services/container';
+
+export default function* startProcessing() {
+  const pangeaLibs = yield call(getPangeaLibrary);
+  while (true) {
+    yield pangeaLibs.queue.txQueue.startProcessing();
+  }
+}
+
```

Comment:
Are you gonna write tests some time later? It would be great if we write tests as we go.

Reply:
Yes, you're right, I'll add tests.